commands/module.ts

```diff
@@ -1,20 +1,28 @@
-// @ts-nocheck
-const marketplaceValidate = require('./module/marketplace-validate');
-const {
-  addConfigOptions,
-  addAccountOptions,
-  addGlobalOptions,
-} = require('../lib/commonOpts');
+import { Argv } from 'yargs';
+import marketplaceValidate from './module/marketplace-validate';
+import { CommandModuleBucket } from '../types/Yargs';
+import { makeYargsBuilder } from '../lib/yargsUtils';
 
-exports.command = 'module';
-exports.describe = false;
+const command = 'module';
+const describe = undefined;
 
-exports.builder = yargs => {
-  addConfigOptions(yargs);
-  addAccountOptions(yargs);
-  addGlobalOptions(yargs);
+type ModuleArgs = unknown;
 
+function moduleBuilder(yargs: Argv): Argv<ModuleArgs> {
   yargs.command(marketplaceValidate).demandCommand(1, '');
+  return yargs as Argv<ModuleArgs>;
+}
 
-  return yargs;
+const builder = makeYargsBuilder<ModuleArgs>(moduleBuilder, command, describe, {
+  useGlobalOptions: true,
+  useConfigOptions: true,
+  useAccountOptions: true,
+});
+
+const moduleCommand: CommandModuleBucket<unknown, ModuleArgs> = {
+  command,
+  describe,
+  builder,
 };
+
+export default moduleCommand;
```

Comment:
Does this also need a default `module.export`? You can test with `hs module --help` I think